timewarp_vk/utils/hmd.hpp

```diff
@@ -0,0 +1,58 @@
+#ifndef _HMD_H
```

Comment:
Please add detailed comments like why we needed this addition. How is this used etc. Similar to what I have said in otehr parts of the code

Reply:
That's just equivalent to `#pragma once`, which prevents double inclusion.